index.html

```diff
@@ -763,9 +762,9 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
       </ol>
     </section>
 
-    <section><h3>The <dfn>create interaction data</dfn> algorithm</h3>
+    <section><h3>The <dfn>create interaction request</dfn> algorithm</h3>
       For a given <a>ConsumedThing</a> object |thing:ConsumedThing|, in order to
-      <a>create interaction data</a> given |source|, |form:Form| and
+      <a>create interaction request</a> given an {{InteractionInput}} as |source|, |form:Form| and
```

Comment:
|source| needs to be kept, since it's used as a variable in the algorithm. The way to express its type is
`|source: InteractionInput|`